app/Transformers/UserCompactTransformer.php

```diff
@@ -48,6 +48,7 @@ public function transform(User $user)
             'is_supporter' => $user->isSupporter(),
             'last_visit' => json_time($user->displayed_last_visit),
             'pm_friends_only' => $user->pm_friends_only,
+            'user_gamemode_rank' => $user->statistics($user->getPlaymodeAttribute($user->osu_playmode), 'true')->rank,
```

Comment:
`statistics` can return `null`

Reply:
`$user->playmode`

Reply:
`'true'` is a string not boolean; also this ends up being `false` in the statistics conditional

Reply:
this ends up being the rank for the user's selected default game mode, so the property name sounds misleading.